lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1237,6 +1241,10 @@ def _create_node_runbook(
                     plan_publisher=plan_publisher,
                 )
 
+        azure_node_runbook.osdisk_size_in_gb = os_disk_size
```

Comment:
The logic is not straightforward. It should make the os_disk_size right, then assign to another variable. BTW, if the default value is set to 10, does it need to check here again?

```python
        if os_disk_size < 10:
            os_disk_size = 10
        azure_node_runbook.osdisk_size_in_gb = os_disk_size
```

Reply:
it will check every image/vhd size, if it is too small, will set it as 10 or 30, otherwise use its default size, I tried if the os disk size is 30, when set it as 10 in template, threw exception during deployment. so just hard code doesn't work.